mlflow/ml-package-versions.yml

```
@@ -42,6 +42,17 @@ pytorch:
     run: |
       pytest tests/pytorch/test_pytorch_model_export.py tests/pytorch/test_pytorch_metric_value_conversion_utils.py
 
+torch:
+  package_info:
+    pip_release: "torch"
+    install_dev: |
+      export PACKAGE_NAME=pytorch
+      pip install git+https://github.com/PytorchLightning/pytorch-lightning.git
+  autologging:
+    minimum: "1.4"
+    maximum: "1.14"
+  run: pytest tests/pytorch/test_tensorboard_autolog.py
```

```suggestion
    run: pytest tests/pytorch/test_tensorboard_autolog.py
```
Good catch @harupy !!